client/src/components/graph/graph.js

```diff
@@ -251,10 +253,49 @@ class Graph extends React.Component {
     };
   }
 
+  // Create image URL based on width and height
+  createImageUrl(width, height) {
+    // larger scale factors may cause the image request to timeout resulting
+    // in no image being shown. Use Chrome Dev Tools to check requests.
+    return globals.API.prefix + globals.API.version + `spatial/image?view_x=${width}&view_y=${height}&scale_factor=1`;
+  }
+
```

Comment:
cool! :) Can you document `scale_factor` better please? Is it the `20x`, `40x` thing? Why would it time-out?